### PR TITLE
saved the secrets

### PR DESCRIPTION
**Pull request by roboMoose**
fingers crossed...

### CODE REVIEW

**Comment by roboMoose** [comment]
okay, so that clearly didnt work...

**Comment by roboMoose** [comment]
huzzah!

File: server/utils/commonUtils.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const crypto = require('crypto');
 
 function hashPassword(password) {
-  const secret = process.env.HASH_SALT || "I'm a teapot";
+  const secret = "I'm a teapot";
   return crypto
```
Comment (nicknapoli82) [inline server/utils/commonUtils.js:5]: I think we still want this to be set to an env var. Looks like its a constant right now?

**Review by nicknapoli82 — COMMENTED** [review]
Aside from my lone comment on the 'I'm a teapot' constant I think things are ok.

File: server/utils/commonUtils.js (hunk 1 above)
Comment (roboMoose) [reply server/utils/commonUtils.js:5]: you are absolutely right - dont know how that got dropped. Pushing it up now

**Review by nicknapoli82 — APPROVED** [review]
Ok. This makes sense to me man. 
Thanks for handling that.

**Pull request merged**